elixir/apps/domain/lib/domain/resources.ex

```diff
@@ -280,22 +275,12 @@ defmodule Domain.Resources do
 
           broadcast_resource_events(:update, resource)
         end,
-        after_replace_commit: fn {replaced_resource, created_resource}, _changesets ->
-          replaced_resource = Repo.preload(replaced_resource, :policies)
-
-          :ok =
-            Enum.each(replaced_resource.policies, fn policy ->
-              {:replaced, _replaced_policy, _created_policy} =
-                Policies.update_or_replace_policy(
-                  policy,
-                  %{resource_id: created_resource.id},
-                  subject
-                )
-            end)
-
-          {:ok, _flows} = Flows.expire_flows_for(replaced_resource, subject)
-          :ok = broadcast_resource_events(:delete, replaced_resource)
-          :ok = broadcast_resource_events(:create, created_resource)
+        after_breaking_update_commit: fn updated_resource, _changeset ->
+          # TODO: Does this still need to happen?
```

Comment:
```suggestion
          # The :delete resource event broadcast is a no-op.
          # This is used to reset the resource on the client and gateway in case filters, conditions, etc are changed.
```